base/applications/games/spider/CMakeLists.txt

```diff
@@ -13,6 +7,7 @@ list(APPEND SOURCE
 add_rc_deps(rsrc.rc ${CMAKE_CURRENT_SOURCE_DIR}/spider.ico)
 add_executable(spider ${SOURCE} rsrc.rc)
 target_link_libraries(spider cardlib)
+target_include_directories(spider PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Because it's a better idea to use the target_*** functions where possible, and I don't want to make specific commits for this, nor do I want to review the whole tree.

So I do it when I edit the files.